yarelismartin/skill-swap
# completed setup

## Description
- Updatated the env. 
- Completed the to-do for this package. 
- Updated package.json with the name of the app and author
- Added the name of my app to the title tag
- Added sample data for post and categories.

## Related Issue
#17 

## Motivation and Context
Essential to complete before starting project. 
Needed in order to display posts when working on CRUD. 

## How Can This Be Tested?
When you pull branch you should be able to login to the app and see 'Skill Share' in the title of the tab. 
You should be able to view the data in the util folder under data. 

## Screenshots (if appropriate):

## Types of changes

[ ] Bug fix (non-breaking change which fixes an issue)
[x ] New feature (non-breaking change which adds functionality)
[ ] Breaking change
